### ISSUE TITLE
[BUG] [Java Service]Return 520 when setting collectSource=true in GET API /i18n/api/v2/translation/products/{productName}/versions/{version}/locales/{locale}/components/{component}/keys/{key}

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#1797]
**Describe the bug**
commit: 5f9c93ce435a14118373688d45c3f27d5b2d3f7a
Return 520 when setting collectSource=true in GET API /i18n/api/v2/translation/products/{productName}/versions/{version}/locales/{locale}/components/{component}/keys/{key}

**To Reproduce**
1. Collect source with below arguments: `https://<ip>:8090/i18n/api/v2/translation/products/VMwareVIP2018/versions/1.1.0/locales/zh-Hans/components/Test.ABC_abc-123/keys/new.key?source=The%20New%20Notes&collectSource=true`
2. See the response:
```
{
  "response": {
    "code": 520,
    "message": "null",
    "serverTime": ""
  },
  "signature": "",
  "data": null
}
```

**Expected behavior**
Should return 200 and new key has been collected.

**Comment by tigershi** [comment]
I have tested singleton that built by latest master branch code, It doesn't appear.

**Comment by lyiyu66** [comment]
Same issue is found on V1:
GET
[​/i18n​/api​/v1​/translation​/product​/{productName}​/key​/{key}]
GET
[​/i18n​/api​/v1​/translation​/product​/{productName}​/component​/{component}​/sources]
GET
[​/i18n​/api​/v1​/translation​/string]
GET
[​/i18n​/api​/v1​/translation​/product​/{productName}​/component​/{component}​/key​/{key}]

**Comment by lyiyu66** [comment]
The issue is fixed with commit 93ffa26a7c0616ff7fee3eb25318342d1dc8975b
